presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -2759,14 +2759,14 @@ velox::core::PlanFragment VeloxBatchQueryPlanConverter::toVeloxQueryPlan(
     // TODO - Use original plan node with root node and aggregate operator
     // stats for additional nodes.
     auto broadcastWriteNode = std::make_shared<operators::BroadcastWriteNode>(
-        "broadcast-write",
+        fmt::format("{}.bw", partitionedOutputNode->id()),
```

Comment:
Can we use the full-form, it will be easier to read? `{}.broadcastWrite` or `{}.broadcast-write`

Reply:
We could, but it takes up space without adding much value. Notice that it makes Plan Node ID column in the screenshot above very wide.